backend/videoService/vonageVideoService.ts

```diff
@@ -59,6 +67,37 @@ class VonageVideoService implements VideoService {
     await this.vonageVideo.stopArchive(archiveId);
     return 'Archive stopped successfully';
   }
+
+  async enableCaptions(sessionId: string): Promise<EnableCaptionResponse> {
+    const requestToken = this.generateToken(sessionId);
+    const { token } = requestToken;
+
+    try {
+      const captionOptions: CaptionOptions = {
```

Comment:
Note: if you notice that `languageCode` is missing here compared to `opentokVideoService`, no you didn't 
when I was adding the caption options,  I found an issue with `vonage-node-sdk` and I filed a PR to fix it here: https://github.com/Vonage/vonage-node-sdk/pull/993
the big issue is that it accepts `languageCode` as `en-us` and not `en-US`; therefore, making it invalid and I'm not able to successfully enable captions.
for the time being, I decided to skip adding the `languageCode` as `en-US` is the default option anyway. once the DevRel team merges in / deploys the new version that has the change I added, I'll add it back here for consistency with `opentokVideoService`